File: templates/python/instrument_meter_distribution_shift.py
"""instrument_meter_distribution_shift: 测量仪表分布漂移（distribution 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='instrument and metering: distribution shift'):
    return make_template_figure('distribution', seed=2812, title=title, domain='instrument and metering', topic='distribution shift')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/instrument_meter_distribution_shift.m
function fig = instrument_meter_distribution_shift()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('distribution', 2812, 'instrument and metering: distribution shift', 'instrument and metering', 'distribution shift');
end
